src/server/wtr-config.js

```diff
@@ -125,44 +131,17 @@ export class WTRConfig {
 		return Object.keys(config).length && { testFramework: { config } };
 	}
 
-	#getPattern(type) {
-		const pattern = this.#cliArgs.files || this.pattern(type);
-
-		// replace filename wildcards with all filter strings
-		// e.g. If filter is ['button', 'list'], pattern './test/*.test.*' becomes:
-		// [ './test/+(*button*.test.*|*.test.*button*)', './test/+(*list*.test.*|*.test.*list*)' ]
-		if (this.#cliArgs.filter) {
-			return this.#cliArgs.filter.map(filterStr => {
-				// replace everything after the last forward slash
-				return pattern.replace(/[^/]*$/, fileGlob => {
-					// create a new glob for each wildcard
-					const fileGlobs = Array.from(fileGlob.matchAll(/(?<!\*)\*(?!\*)/g)).map(({ index }) => {
-						const arr = fileGlob.split('');
-						arr.splice(index, 1, `*${filterStr}*`);
-						return arr.join('');
-					});
-					return `+(${fileGlobs.join('|')})`;
-				});
-			});
-		}
-		return pattern;
-	}
-
 	create({
 		pattern = DEFAULT_PATTERN,
-		vdiff = DEFAULT_VDIFF,
 		timeout,
 		...passthroughConfig
 	} = {}) {
 
-		const { files, filter, golden, grep, group, manual, playwright, watch } = this.#cliArgs;
+		const { files, filter, golden, grep, group, manual, watch } = this.#cliArgs;
+		const passthroughGroupNames = passthroughConfig.groups?.map(g => g.name) ?? [];
 
-		if (!group || group === 'default') {
-			if (playwright) {
-				console.warn('Warning: reducedMotion disabled. Use the unit group to enable reducedMotion.');
-			} else {
-				console.warn('Warning: Running with puppeteer, reducedMotion disabled. Use the unit group to use playwright with reducedMotion enabled');
-			}
+		if (!['test', 'vdiff', ...passthroughGroupNames].includes(group)) {
+			return {}; // allow wtr to error
 		}
 
 		delete passthroughConfig.browsers;
```

Comment:
Do we still need this if we never send this along? Or is this in case they set `browsers` in their config file?

Reply:
Yeah, the latter.